apps/server/src/controllers/userControllers/authUsers.ts

```diff
@@ -360,3 +362,68 @@ export const resetPassword = async (req: Request, res: Response) => {
     })
   }
 }
+
+export const changePassword = async (req: RequestExt, res: Response) => {
+  const passwordRegex = passwordUtils.regex
+
+  try {
+    const { _userId: userId, ...rest } = req.body
+
+    const requestData = changePasswordSchema.strict().safeParse(rest)
+
+    if (!requestData.success) {
+      return res.status(HTTP_STATUS_CODE.BAD_REQUEST).json({
+        message: requestData.error.issues,
+      })
+    }
+
+    const _data = requestData.data
```

Comment:
usually, I use the underscore variable when there's an existing scope with the same name that may be confusing to read